portfolio/pom.xml

```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
```

Comment:
Remove the TODO now that it's done.

Reply:
Got it, fixed it in the following commit.